auxiliary/auxiliary.go

```diff
@@ -13,6 +14,85 @@ import (
 	"golang.org/x/crypto/ssh/terminal"
 )
 
+// ParseArgs parses the command line arguments and returns the
+// user's configuration options for the caller to then use.
+func ParseArgs(args []string) (string, error) {
+	helpMessage := fmt.Sprintf(`Usage: %s [arguments]
```

Comment:
Huh, my bad. Thought the assignment at https://github.com/libeclipse/pocket/pull/5/files/df5340d7624ddf97e30776a2a13fdc811bbe66d8#diff-14c7fbe3b9846641332efbe6310855bdR26 was arg[1]